griptape/utils/observability/griptape_cloud_span_exporter.py

```diff
@@ -0,0 +1,49 @@
+import os
+from collections.abc import Sequence
+from urllib.parse import urljoin
+from typing_extensions import override
+from attrs import Factory, define, field
+from opentelemetry.sdk.trace import ReadableSpan
+from opentelemetry.sdk.trace.export import SpanExporter, SpanExportResult
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter
+
+
+@define
+class GriptapeCloudSpanExporter(SpanExporter):
```

Comment:
This class seems suitable to make into a Driver -- even if it's not a traditional one. For instance, what if I wanted to connect to [Prometheus](https://opentelemetry.io/docs/languages/js/exporters/#prometheus) instead?

Reply:
Added drivers!